src/incubator/gradient/types.ts

```diff
@@ -0,0 +1,35 @@
+import type {LinearGradientProps} from 'react-native-linear-gradient';
+
+type CommonGradientProps = Pick<LinearGradientProps, 'colors' | 'children'> & {
+  angle?: number;
+  center?: boolean;
```

Comment:
The center alignment props only affect the children content, not the gradient.
These should be moved to a separate prop group like `ContentAlignmentProps` with boolean values, WDYT ?

Reply:
Doesn't `center` always affects the children?

Reply:
it does, for me when I pass `center` or `centerV/H` to the gradient I expect the color of the gradient to be aligned also.
I leave it up to you.

Reply:
I've changed the screen's text but I think the modifier is ok